extra/3RD_PARTY_QUICK_START.md

```diff
@@ -9,56 +9,69 @@ Take a look at some of the real-life examples first:
+
+Set your repo directory structure to follow exactly the example(s) above (the template repository does this for you, so if you started with the template you can skip this step.)
+
+For example, if your grammar is named `your-language`, create your repository directory structure as follows (rename `your-language` to match your language name. For example, if your language is `pascal`, then rename all occurrences of `your-language` with `pascal`):
+
+- Put your language file in `src/languages/your-language.js`.
+- Add detect tests in `test/detect/your-language`.
+- Add markup tests in `test/markup/your-language`.
+- Add a `package.json` file.
+- Add a `dist` folder (see [Packaging](#packaging), below.)
+- Include a license.
+- Include a README to help users install and use your language with highlight.js.
 
 ## Testing
 
+Going back to your clone of `highlight-js` core repo, `git clone` or symlink your language repo into the `extra` folder. There should be an `extra/your-language` folder for your language.
+
+> Any 3rd party language files placed here should not be committed to the highlight-js repository (by default they are ignored, just don't override that behavior.)
+
 To test (detect and markup tests), just build highlight.js and test it.  Your tests should be automatically run with the suite:
 
-```
+```bash
 node ./tools/build.js -t node
 npm run test
 ```
 
-If you can't get the auto-detect tests passing you should simply turn off auto-detection for your language in its definition with `disableAutodetect: true`.  Auto-detection is hard.
-
+If you can't get the auto-detect tests passing then turn off auto-detection for your language in its definition with `disableAutodetect: true`.  Auto-detection is hard.
 
 ## Packaging
 
-Users will expect your package to include a minified CDN distributable in your `dist` folder. This should allow them to add the module to their website with only a single `<script>` tag.
+Users will expect your package to include a minified CDN distributable in your `dist` folder. This allows them to add the module to their website with only a single `<script>` tag.
 
-Luckily, the highlight.js CDN build process will build this file for you automatically.  You can simply commit it to your repo, and done.
+The highlight.js CDN build process will build this file for you automatically. You can simply commit and push your repo, and done.
 
-```
+```bash
 node ./tools/build.js -t cdn
 
 ...
-Building extra/highlightjs-xyz/dist/xyz.min.js.
+Building extra/highlightjs-your-language/dist/your-language.min.js.
 ...
 ```
 
+Copy `extra/highlightjs-your-language/dist/your-language.min.js` to your repo's dist folder `{your-language-repo}/dist/your-language.min.js`, commit and push it.
```

Comment:
Why?  There should be nothing to copy.  If their repo is under extra then the file is already in the dist folder exactly where it belongs, it only needs to be committed.